src/env.d.ts

```diff
@@ -0,0 +1,7 @@
+
+/**
+ * Intellisense for Typescript
+ */
+interface ImportMetaEnv {
+  VITE_API_URL: string;
```

Comment:
Let's add a JSDocs here

Reply:
done

Reply:
I can't see updates

Reply:
I've updated it

Reply:
I suppose, @neSpecc suggests adding comment describing `VITE_API_URL` purpose

Reply:
noted @TatianaFomina

Reply:
@TatianaFomina I've updated, I don't know if @neSpecc meant otherwise